libcore/src/pedestrian/AgentsSource.cpp

```diff
@@ -73,7 +73,7 @@ AgentsSource::~AgentsSource() {}
 void AgentsSource::GenerateAgentsAndAddToPool(int count, Building * building)
 {
     std::vector<Pedestrian *> peds;
-    GenerateAgents(peds, count, building);
+    GenerateAgents(peds, count, building); // positions are not set yet!
```

Comment:
Could you add this comment as comment for GenerateAgents in the header file?

Reply:
ok